articles/how-to-carry-out-effective-user-acceptance-testing-uat/index.md

```diff
@@ -0,0 +1,77 @@
+
+UAT is different from [unit testing](https://www.tutorialspoint.com/software_testing_dictionary/unit_testing.htm), [integration testing](https://www.guru99.com/integration-testing.html), and [system testing](https://www.softwaretestinghelp.com/system-testing/). System testing ensures that the system does not crash and meets the documented requirements. On the other hand, UAT ensures that the software system is working as per the business requirements. The system is evaluated in real-life scenarios from the external user point of view. You should do the test in an environment equivalent to a production environment.
+
+The diagram below shows where UAT falls in the software development process.
+
+![software process](/engineering-education/how-to-carry-out-effective-user-acceptance-testing-uat/software-process.jpg)
+
+#### Objectives of UAT
+
+- To confirm that the software product can perform all the business functions.
+- To confirm that the software product is usable from the end-users perspective.
+- To certify that the software product is ready to move to production.
+
+#### Who conducts UAT?
+
+The end-users or the client conducts UAT. These can be the end-users of a custom made software or someone who has requested custom software from a software developer as a service. The people performing the test are called the test team. In an organization, the team should comprise members from all departments and different groups. The testing team should comprise of subject matter experts. For example, accounting software UAT should involve accountants.
+
+#### UAT Prerequisites
+
+Some requirements must be met before performing a UAT test. These include:
+
+- Provide business requirements to the testing team. UAT is done against business requirements.
+- The system must be fully developed and error-free. Only cosmetic defects are allowed. These are low-priority errors from a business point of view and do not affect the testing process.
+- System testing, integration testing, and unit testing should be completed. System errors are not expected at this stage. The reported flaws are already fixed and evaluated before UAT.
+- UAT environment should be available.
+- [Regression testing](https://www.guru99.com/regression-testing.html) should be complete to confirm that recent code changes do not affect the software.
+
+### Why you need user acceptance testing
+
+The main reason why you need user acceptance testing is to validate if all business requirements of the software are met. This must be done before releasing the software to the market.
+
+UAT also validates that change requirements are implemented tou suit the business requirements. There could be some business changes that might not have been communicated to the development team effectively. UAT would be able to capture such.
```

Comment:
@Tsanguu Thank you for working on the suggestions. I just noticed a minor error in the content you added. It is 'implemented to' not 'implemented tou'. Please change it-im not able to do so from my end.

Reply:
@mbaabuones corrected. That must have been a stray character.